tools/creality_printer_test_server.py

```
@@ -888,22 +1323,125 @@ async def on_track(track):
+            wanted_rtx = [c for c in codecs if is_rtx(c)
+                          and c.parameters.get("apt") in wanted_pts]
+            keep = {id(c) for c in wanted + wanted_rtx}
+            rest = [c for c in codecs if id(c) not in keep]
+
+            transceiver._codecs = wanted + wanted_rtx + rest
+            LOGGER.debug(
+                "video codecs reordered -> %s",
+                ", ".join(f"{c.mimeType}/{c.payloadType}" for c in transceiver._codecs[:4]),
+            )
+
     async def _cleanup_pc(self, pc: RTCPeerConnection, sink: MediaBlackhole):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_cleanup_pc`

Add return type annotation: `None`

(ANN202)